New alien faction: the Bunrodea (#6438)
endless-sky/endless-sky: data/map.txt

			distance 545
			period 35.9174

system "Paru Paru"
	pos 275.9 -237.3
	government Bunrodea
	attributes "neutron"
	habitable 100
	belt 1500
	link "Avo Chigo"
	link "Eragaru Le"
	link "Ravu Kon"
	asteroids "small rock" 59 3.3852
	asteroids "medium rock" 4 2.184
	asteroids "large rock" 30 3.5308
	asteroids "small metal" 8 3.6764
	asteroids "medium metal" 10 3.4216
	asteroids "large metal" 11 5.1688
	trade Clothing 290
	trade Electronics 740
	trade Equipment 530
	trade Food 350
	trade "Heavy Metals" 960
	trade Industrial 720
	trade "Luxury Goods" 1220
	trade Medical 680
	trade Metal 390
	trade Plastic 390
	fleet "Bunrodea Erabu Freight" 900

> 	fleet "Bunrodea Defense" 2000

Reviewer: ```suggestion
	fleet "Bunrodea Defense" 2000
	hazard "Neutron Star Magnetic Field" 1
```
now that we have it, it should be added here too.

			distance 273.55
			period 41.1893
	object
		sprite planet/rock17-b
		distance 2309.96
		period 984.429

system "Sera Natta"
	pos 344.9 -418.3
	government Bunrodea
	habitable 3000
	belt 1500
	link "Urba Pest"
	asteroids "small rock" 81 1.7472
	asteroids "medium rock" 6 1.5561
	asteroids "large rock" 7 3.0303
	asteroids "small metal" 8 2.9757
	asteroids "medium metal" 35 2.3478
	asteroids "large metal" 8 1.4196
	trade Clothing 290
	trade Electronics 740
	trade Equipment 530
	trade Food 350
	trade "Heavy Metals" 960
	trade Industrial 720
	trade "Luxury Goods" 1220
	trade Medical 680
	trade Metal 390
	trade Plastic 390
	fleet "Bunrodea Defense" 3000
	object

> 		sprite star/k-giant

Reviewer 2: Giving the system a bluer star, the better to cast UV light with.
```suggestion
		sprite star/a8
```

	spaceport `In addition to the usual services you would expect, the spaceport of Deli Kasi is also home to a number of genetic engineering labs geared toward the creation of more bountiful plant seeds. It's unsurprising that the Bunrodeans would genetically modify their food, but for as much as the Bunrodeans are capable of, nothing beats the usefulness of fertile farm land like can be found on this planet.`
	"required reputation" 10
	bribe 0
	security 0

planet "Deli Kat"
	attributes bunrodea farming
	landscape land/fields0
	description `Deli Kat orbits the dimmest star out of any of the inhabited systems in this region of space, a star not too disimilar to Sol. Although the planet orbits farther from its host star than Earth does, its slightly thicker atmosphere seems to balance that out, resulting in a mostly Earth-like climate. The planet's many vast plains and river systems additionally help to make Deli Kat a fertile farming world.`

> 	spaceport `The spaceport of Deli Kat is in a remote location where the coming and going of spaceships is unlikely to disturb any livestock. Goods are brought into the spaceport via high speed train-like vehicles, except without the need to travel on rails thanks to the use of technology that looks similar to the repulsor engines on human ships. Deli Kat's flat terrain makes it easy for these amphibious vehicles to bullet across the surface, picking up supplies from farms and beinging them to the spaceport for transportation off-world.`

Author: I'd prefer to use bullet as a verb than to create a simile.
![image](https://user-images.githubusercontent.com/17688683/204105097-38f3dd4a-5f8c-4fbb-a1f5-e752faa0ae77.png)

	bribe 0.1
	security 0.5

planet Okoity
	attributes bunrodea urban
	landscape land/fog2
	description `Okoity has the highest population of any Bunrodean world, with a staggering 24 billion people. All but the most inhospitable regions of the planet have been settled in some shape or form.`
	description `	The planet saw a large influx of people with the destruction of Olohoa's atmosphere in the nearby system of Sera Natta, massive even relative to the already preexisting population. Although this occurred about a century ago, the planet still seems to be struggling with this increased population.`
	spaceport `The spaceport of Okoity is completely packed with Bunrodeans. You have little room to maneuver as you explore the spaceport while trying to not bump into anyone. Many of the Bunrodeans here show the signs of being genetically engineered, from Bunrodeans with extra limbs to one that you see up high climbing on walls and jumping from building to building to avoid the crowd below.`
	shipyard "Bunrodea Basic"
	outfitter "Bunrodea Basic"
	"required reputation" 25
	bribe 0
	security 0

planet Olohoa
	attributes bunrodea uninhabited
	landscape land/sky3

> 	description `From afar, Olohoa looks like it might be a pleasant place to live, with clear oceans not too disimilar to what you might find on a Paradise World. But your ship's sensors tell you a different story, one of a planet with immense amounts of ultraviolet radiation bombarding its surface. Nothing short of an extremophile could survive on the surface here, and any life in the oceans is likely deep down where no radiation can reach.`
> 	description `	As you explore, you begin to find occasional signs of civilization in the form of small cities with landing pads for ships, but these areas are just as lifeless as the rest of the planet.`

Reviewer 2: Making this system just naturally barren

```suggestion
	description `From afar, Olohoa looks like it might be a pleasant place to live, with clear oceans not too disimilar to what you might find on a Paradise World, were you to ignore the local star. But your ship's sensors tell you a different story, one of a planet with immense amounts of ultraviolet radiation bombarding its surface. Nothing short of an extremophile could survive on the surface here, and any life in the oceans is likely deep down where no radiation can reach.`
```

	bribe 0
	security 0

planet Sardva
	attributes bunrodea religious
	landscape land/fields6
	description `The planet of Sardva is used by the Bunrodea as a vault for genetic material of all origins, from the seeds of plants that the Bunrodea have encountered to genetic material of animals and even material from past generations of Bunrodeans. This material is stored in many of the planet's cave systems, which, with some slight modification, have amazingly stable climates that are perfect for the long term storage of genetic material.`
	description `	The decentralized and redundant nature of the vaults also ensures that if anything happens on one part of the planet that may compromise the vaults there, locations elsewhere on the planet will still keep the material safe.`

> 	spaceport `Sardva has no single major spaceport, with each vault location supporting its own small spaceport for receiving shipments and refueling the freighters. You eventually find a vault that is willingly to let you land so that you may refuel, but they don't seem too happy about your presence, and seem like they're only refueling you so that you stop asking every vault you come across.`

Reviewer 3: ```suggestion
	spaceport `Sardva has no single major spaceport, with each vault location supporting its own small spaceport for receiving shipments and refueling the freighters. You eventually find a vault that is willing to let you land so that you may refuel, but they don't seem too happy about your presence, and seem like they're only refueling you so that you stop asking every vault you come across.`
```

	landscape land/desert7
	description `This was probably a habitable world at one point. It is no longer. Nothing lives or grows here, and the atmosphere is full of toxic chemicals.`

planet Schisathro
	attributes bunrodea guild manufacturing mining
	landscape land/canyon5

> 	description `Schisathro is a major mining world, and also home to the largest shipyards in Bunrodean space. Larger shipyards than the ones found here only exist on the capital world of Erabuthro.`

Reviewer 3: ```suggestion
	description `Schisathro is a major mining world, and also home to some of the largest shipyards in Bunrodean space. Larger shipyards than the ones found here only exist on the capital world of Erabuthro.`
```

Author: No, these *are* the largest.

Reviewer 4: They can't be the largest, because you're saying that larger ones exist on Erabuthro.

Author: True.

> 	description `	The planet's success as a mining world can be owed to its many deep canyons found across most of the planet's above water surface, which allow for easier access to mineral rich deposits in the crust. The major shipyards developed alongside these mines, allowed for easy transportation of material directly to the shipyards instead of requiring lengthy journeys through space.`

Reviewer 3: ```suggestion
	description `	The planet's success as a mining world can be owed to its many deep canyons found across most of the planet's above water surface, which allow for easier access to mineral rich deposits in the crust. The major shipyards developed alongside these mines allowed for easy transportation of material directly to the shipyards instead of requiring lengthy journeys through space.`
```

	bribe 0
	security 0

planet Sopoyra
	attributes bunrodea manufacturing shipping
	landscape land/industrial0-iridium

> 	description `Sopoyra is a major manufacturing planet, as well as the chokepoint between the Megasa systems of the north and the Erabu systems of the south. Given that royal permission is required to enter the Erabu region, this world also acts as the focal point for trade between the the two regions. Goods are dropped off from other Megasa worlds, then transported to the Erabu worlds by freighters with royal permission.`

Reviewer 3: ```suggestion
	description `Sopoyra is a major manufacturing planet, as well as the chokepoint between the Megasa systems of the north and the Erabu systems of the south. Given that royal permission is required to enter the Erabu region, this world also acts as the focal point for trade between the two regions. Goods are dropped off from other Megasa worlds, then transported to the Erabu worlds by freighters with royal permission.`
```

> 	spaceport `Sopoyra is not a very hospitibale world to life, so there are few settlements outside of the main spaceport, which is almost entirely dedicated to factories and warehouses.`

Reviewer 3: ```suggestion
	spaceport `Sopoyra is not a very hospitable world. There are few settlements outside of the main spaceport, which is almost entirely dedicated to factories and warehouses.`
```

> 	spaceport `	There are freighters that will come from the north to drop off goods here, only for them to be immediately picked up by another freighter that brings the good down south. It seems somewhat inefficient to do things this way, but must be done to be in keeping with royal decrees.`

Reviewer 3: ```suggestion
	spaceport `	There are freighters that will come from the north to drop off goods here, only for them to be immediately picked up by another freighter that brings the goods down south. It seems somewhat inefficient to do things this way, but must be done to be in keeping with royal decrees.`
```
